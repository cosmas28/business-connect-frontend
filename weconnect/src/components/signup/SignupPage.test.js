import React from 'react';
// import TestUtils from 'react-addons-test-utils';
import ReactDOM from 'react-dom';
import SignUpPage from './SignupPage';

describe('Sign up Page', () => {
    it('renders signup page without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<SignUpPage />, div);
        ReactDOM.unmountComponentAtNode(div);
    });
});
