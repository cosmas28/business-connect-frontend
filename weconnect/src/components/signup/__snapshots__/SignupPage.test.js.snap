// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sign up Page renders signup page without crashing 1`] = `
<main
  className="main-content"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <Home />
      <SignUpForm
        handleInputChange={[Function]}
        handleSubmitForm={[Function]}
        outPutMessage=""
      />
    </div>
  </div>
</main>
`;
